Source/BusinessMomentsAdaptorImage/server.js

```diff
@@ -0,0 +1,28 @@
+// Require express and body-parser
```

Comment:
@parhambarazesh It seems to me that this js file can be delete now as the typescript file is now working?

Reply:
Not sure yet. If we remove server.js, then should I include the "tsc server.js" command in the docker file? or creating the js file is handled somehow in the pipeline?